NetTopologySuite/IO/WKBReader.cs

```diff
@@ -245,12 +224,12 @@ protected IGeometry Read(BinaryReader reader)
         /// 
         /// </summary>
         /// <param name="reader"></param>
-        private void ReadByteOrder(BinaryReader reader)
+        private void ReadByteOrder(ConfigurableBinaryReader reader)
         {
             reader.ReadByte();

```

Comment:
Ahh, when I'd made my original comment, I meant that it seems like at least this line should change to set `reader.EncodingType` here, since the PostGis reader's version would do the same.